pytorch_lightning/lite/wrappers.py

```diff
@@ -164,6 +164,11 @@ def __init__(self, dataloader: Iterable, device: Optional[torch.device] = None)
         self._dataloader = dataloader
         self._device = device
 
+    def __len__(self) -> Union[int, float]:
+        if isinstance(self._dataloader, Sized):
+            return len(self._dataloader)
+        return float("inf")
```

Comment:
This does not belong in this PR. Why did we add this? 
Needs to be addressed in #10297